packages/core/core/src/factories/useUserShippingFactory.ts

```diff
@@ -40,20 +40,26 @@ export const useUserShippingFactory = <ADDRESS>(
     const readonlyDefaultAddress: Readonly<ADDRESS> = unref(defaultAddress);
 
     const addAddress = async (address: ADDRESS) => {
+      Logger.debug('useUserShipping.addAddress', address);
```

Comment:
probably the same comment about paswords applies here - should pii data be masked to prevent accidental leaking